.circleci/verbatim-sources/job-specs/pytorch-job-specs.yml

```diff
@@ -31,7 +31,7 @@ jobs:
           fi
           echo "Parallel backend flags: "${PARALLEL_FLAGS}
 
-          export COMMAND='((echo "export BUILD_ENVIRONMENT=${BUILD_ENVIRONMENT}" && echo '"$PARALLEL_FLAGS"' && echo "source ./workspace/env" && echo "sudo chown -R jenkins workspace && cd workspace && .jenkins/pytorch/build.sh") | docker exec -u jenkins -i "$id" bash) 2>&1'
+          export COMMAND='((echo "export BUILD_ENVIRONMENT=${BUILD_ENVIRONMENT}" && echo '"$PARALLEL_FLAGS"' && echo "source ./workspace/env" && echo "sudo chown -R jenkins workspace && cd workspace && .jenkins/pytorch/build.sh && find /var/lib/jenkins/workspace -type f -name "*.a" -exec rm {} \; && find /var/lib/jenkins/workspace -type f -name "*.o" -exec rm {} \;") | docker exec -u jenkins -i "$id" bash) 2>&1'
```

Comment:
Same as above:
```suggestion
          export COMMAND='((echo "export BUILD_ENVIRONMENT=${BUILD_ENVIRONMENT}" && echo '"$PARALLEL_FLAGS"' && echo "source ./workspace/env" && echo "sudo chown -R jenkins workspace && cd workspace && .jenkins/pytorch/build.sh && find /var/lib/jenkins/workspace/build -type f -name "*.a" -or -name "*.o"|xargs rm") | docker exec -u jenkins -i "$id" bash) 2>&1
```

Reply:
This one is autogenerated by .circleci/regenerate.sh